csharp-selenium-webdriver-sample/SampleTest.cs

```diff
@@ -6,57 +6,72 @@
 using System;
 using System.IO;
 using Selenium.Axe;
-using OpenQA.Selenium.Chrome;
 
 namespace CSharpSeleniumWebdriverSample.Test
 {
     [TestClass]
-    [DeploymentItem("SamplePage.html")]
+    //  Install chromedriver to enable Chrome broswer and copy it to the same directory as the deployed test assemblies.
     [DeploymentItem("chromedriver.exe")]
+    //  Install geckodriver to enable Firefox brower and copy it to the same directory as the deployed test assemblies.
     [DeploymentItem("geckodriver.exe")]
     public class SampleTest
     {
+        //  Intance to control the browser.
         private IWebDriver _webDriver;
 
         [TestCleanup]
         public virtual void TearDown()
         {
+            //  You should quit the driver on TearDown which close every associated window.
             _webDriver?.Quit();
         }
 
 
         [TestMethod]
+        // Test on Chrome
         [DataRow(BrowserType.Chrome)]
+        //  Test on Firefox
         [DataRow(BrowserType.Firefox)]
         public void RunScanOnGivenElement(BrowserType browser)
         {
+            //  Intilaize the driver based on the browser type
```

Comment:
I don't think any of the comments from here through line 53 are adding much value; they're all just repeating the code directly underneath them. I'd definitely recommend looking over index.test.ts in the typescript sample for some examples of the types of comments I think would be more useful, describing either *why* particular bits of code are present or describing what's happening at a higher level (eg, whole tests)